core/templates/components/forms/schema-based-editors/schema-based-float-editor.component.spec.ts

```diff
@@ -260,4 +260,16 @@ describe('Schema based float editor component', function () {
     expect(component.localValue).toEqual(null);
     expect(component.errorStringI18nKey).toEqual('Error');
   });
+
+  it('should not show error to diffrent decimal points', () => {
```

Comment:
Spelling: different

(Also: to --> for)

Reply:
Thank you,
pushing changes rn